students/Kenneth_Murray/Lesson2/Grid1/Grid1.py

```diff
@@ -0,0 +1,32 @@
+def grid1(x):
+#This will create a grid with 4 equal quadrants
+#The input must be an odd number 
+#the input is equall to the vertical or horizontal lines that make up the grid
+#The minimum value is 5
+#The input must be an integer
+
+#assign variables to the print characters
+	#horizontal 
+	h="-"
```

Comment:
You comment here just actually just be the variable name.  Avoid single character variable names.